css/_atlaskit_overrides.scss

```diff
@@ -37,30 +20,6 @@
  * Override @atlaskit/theme styling for the top toolbar so it displays over
  * the video thumbnail while obscuring as little as possible.
  */
-.videocontainer .tOoji {
+.videocontainer__toptoolbar > div > div {
     background: none;
 }
-
-/**
- * Override @atlaskit/InlineDialog styling for the overflowmenu so it displays
- * with the correct height.
- */
-.toolbox-button-wth-dialog .eYJELv {
-    max-height: initial;
-}
-
-/**
- * Override @atlaskit/InlineDialog styling for the overflowmenu so it displays
- * a scrollable list of elements at small screen widths.
- */
-.sc-eNQAEJ {
-    overflow-y: auto;
-}
-
-/**
- * Keep overflow menu within screen vertical bounds and make it scrollable.
- */
-.toolbox-button-wth-dialog .sc-ckVGcZ.fdAqDG > :first-child {
```

Comment:
I think I was the one to mess this up, since I've added this recently. I wasn't aware that the styled-components classes would differ between the dev environment and the released ones, but the intent of this selector was to keep the overflow menu within the screen in a narrow landscape scenario. (See attached screenshot)

<img width="954" alt="Screenshot 2021-01-13 at 11 00 48" src="https://user-images.githubusercontent.com/52234168/104887128-755be580-5973-11eb-9a72-0706c6e4f0ce.png">

However I think this could be replaced with a hackier, but functional selector such as:
`.toolbox-button-wth-dialog > div > div:nth-child(2) > div`

Reply:
> I wasn't aware that the styled-components classes would differ between the dev environment and the released ones

I changed the styles to not depend on those generic classes but on the class suffixes that are added by some atlaskit components. After creating a release build I noticed that those suffixes are gone. So I would say the safest way is to style the component based on HTML structure. I will also re-integrate the styles you mentioned above. I wasn't sure what result to expect from those. Thanks for the explanation!